### PR TITLE
DQC_0015 exclusions xlsx and resources files

### PR DESCRIPTION
**Pull request by davidtauriello**
### Changes:

- updated member exclusions and list elements .csv files for Committee-approved changes to DQC_0015
for Committee-approved changes to DQC_0015 (Feb, Apr, May)
- revised member exclusions .xlsx (documentation) based on .csv edits

### Pull request process

- [x] Full description of changes provided above.
- [x] Reviewed by another person. (+1)
- [x] If this is a code change, reviewed by a second person. (+1)
- [ ] QA'd to the specifications listed [here](https://github.com/DataQualityCommittee/dqc_us_rules#quality-assurance-qa-of-a-pull-request). Documentation only changes do not require the formal code quality assurance process. (+10)

##### Once checklist is complete, @hefischer  @andrewperkins-wf please review for merge.

### CODE REVIEW

**Comment by davidtauriello** [comment]
@andrewperkins-wf @hermfischer-wf Not sure why the /resources exclusions .csv has 6 header columns and the rows in this list have only 5 entries - ?  Can the heading column be removed so this file is beautiful and searchable (https://help.github.com/articles/rendering-csv-and-tsv-data) ?

@jwmstarr @lrohman @campbellpryde @amibeers @susanyount-wf - please review.

File: dqc_us_rules/resources/DQC_US_0015/dqc_15_concepts.csv [hunk 1]
```diff
@@ -1,4 +1,4 @@
-200,AcceleratedShareRepurchasesFinalPricePaidPerShare
+200,AcceleratedShareRepurchasesFinalPricePaidPerShare
```
Comment (andrewperkins-wf) [inline dqc_us_rules/resources/DQC_US_0015/dqc_15_concepts.csv]: Should not have a trailing space

File: dqc_us_rules/resources/DQC_US_0015/dqc_15_exclusion_rules.csv [hunk 2]
```diff
@@ -3,19 +3,31 @@ BLE,Member,,Contains_insensitive,Adjust
...
+BLE,Member,,Equals,us-gaap:AccumulatedNetGainLossFromDesignatedOrQualifyingCashFlowHedgesMember
+BLE,Member,,Equals,us-gaap:AccumulatedNetUnrealizedInvestmentGainLossMember
 BLE,Member,,Equals,us-gaap:DeferredDerivativeGainLossMember
 BLE,Member,,Equals,us-gaap:AboveMarketLeasesMember
-BLE,Member,,Equals,us-gaap:MaterialReconcilingItemsMember
 BLE,Member,,Equals,us-gaap:NewAccountingPronouncementEarlyAdoptionEffectMember
-BLE,Member,,Equals,us-gaap:AccumulatedNetGainLossFromDesignatedOrQualifyingCashFlowHedgesMember
-BLE,Member,,Equals,us-gaap:AccumulatedNetUnrealizedInvestmentGainLossMember
+BLE,Member,,Equals,us-gaap:UnallocatedFinancingReceivablesMember
+BLE,Member,,Has_member,us-gaap:StatementBusinessSegmentsAxis|us-gaap:CorporateAndOtherMember
```
Comment (andrewperkins-wf) [inline dqc_us_rules/resources/DQC_US_0015/dqc_15_exclusion_rules.csv]: Second column should be ```Axis``` not ```Member```

File: dqc_us_rules/resources/DQC_US_0015/dqc_15_exclusion_rules.csv [hunk 3]
```diff
@@ -3,19 +3,31 @@ BLE,Member,,Contains_insensitive,Adjust
...
+BLE,Member,,Has_member,us-gaap:StatementBusinessSegmentsAxis|us-gaap:CorporateMember
```
Comment (andrewperkins-wf) [inline dqc_us_rules/resources/DQC_US_0015/dqc_15_exclusion_rules.csv]: Second column should be ```Axis``` not ```Member```

File: dqc_us_rules/resources/DQC_US_0015/dqc_15_exclusion_rules.csv [hunk 4]
```diff
@@ -3,19 +3,31 @@ BLE,Member,,Contains_insensitive,Adjust
...
+BLE,Member,,Has_member,us-gaap:StatementBusinessSegmentsAxis|us-gaap:AllOtherSegmentsMember
```
Comment (andrewperkins-wf) [inline dqc_us_rules/resources/DQC_US_0015/dqc_15_exclusion_rules.csv]: Second column should be ```Axis``` not ```Member```

File: dqc_us_rules/resources/DQC_US_0015/dqc_15_exclusion_rules.csv [hunk 5]
```diff
@@ -3,19 +3,31 @@ BLE,Member,,Contains_insensitive,Adjust
...
+BLE,Axis,,Equals,us-gaap:QuantifyingMisstatementInCurrentYearFinancialStatementsByNatureOfErrorAxis
+BLE,Axis,,Equals,us-gaap:PartnerCapitalComponentsAxis
+BLE,Axis,,Equals,us-gaap:ChangeInAccountingEstimateByTypeAxis
+BLE,Axis,,Equals,us-gaap:PartnerTypeOfPartnersCapitalAccountAxis
 BLE,Axis,,Has_member,dei:LegalEntityAxis|us-gaap:ParentCompanyMember
 BLE,Axis,,Has_member,dei:LegalEntityAxis|us-gaap:SubsidiariesMember
 BLE,Axis,,Has_member,dei:LegalEntityAxis|us-gaap:GuarantorSubsidiariesMember
 BLE,Axis,,Has_member,dei:LegalEntityAxis|us-gaap:NonGuarantorSubsidiariesMember
+BLE,Axis,,Has_member,dei:LegalEntityAxis|us-gaap:SubsidiaryIssuerMember
+BLE,Member,,Has_member,us-gaap:FairValueByMeasurementBasisAxis|us-gaap:ChangeDuringPeriodFairValueDisclosureMember
```
Comment (andrewperkins-wf) [inline dqc_us_rules/resources/DQC_US_0015/dqc_15_exclusion_rules.csv]: Second column should be ```Axis``` not ```Member```

**Comment by andrewperkins-wf** [comment]
pass complete

It has 6 header columns to show that there can be "ands" appended with extra columns.

**Comment by andrewperkins-wf** [comment]
Also, I appreciate you updating the csv file for us @davidtauriello !

**Comment by chrislococo-wf** [comment]
@davidtauriello Could you please update your PR with the template found here: https://github.com/DataQualityCommittee/dqc_us_rules/blob/master/.github/PULL_REQUEST_TEMPLATE.md

Sadly, I don't have permissions to do it for you otherwise I would. 😄

**Comment by davidtauriello** [comment]
@chrislococo-wf - my local copy wouldn't sync to the pull without an edit which has been removed.  Let me know if I can do anything else to help with this update.

**Comment by chrislococo-wf** [comment]
Oh, sorry for the confusion, I meant for you to put the template into the PR description field of this. The changes should already be in master, so no changes are needed to the actual code.

**Comment by davidtauriello** [comment]
@chrislococo-wf - thx for clarification :-)

**Comment by andrewperkins-wf** [comment]
+1

**Comment by jasonleinbach-wf** [comment]
+1

**Comment by jasonleinbach-wf** [comment]
+10 - tested various DQC 15 test cases, and all worked as expected.

**Comment by andrewperkins-wf** [comment]
Reviewed and tested - merging.

**Pull request merged**